yk-saito/hotel-booking-dapp
# 14 add room UI

Fix #14 

## What


Create UI to register rooms
- framework: React Bootstrap

## Test


1. install && start
```shell
yarn && yarn build
```
2. Connect wallet and add room

![01](https://user-images.githubusercontent.com/60546319/179665191-b81e3373-f109-4093-961c-22de2dbbf5e3.jpg)

3. Check console log

![02](https://user-images.githubusercontent.com/60546319/179665123-18179b10-d516-45d7-b796-1c2a073c7680.jpg)


4. Check with `near-cli `
```shell
near view <DEPLOYED_ID> get_rooms
```
Expected
```shell
View call: dev-1658204414506-33098072765600.get_rooms()
[
  {
    room_id: '1658204774622',
    name: 'test',
    image: 'https://www.s-hoshino.com/f_photo/onsen/on_040.jpg',
    description: 'test',
    location: 'test',
    price: '1000000000000000000000000',
    owner: 'ysaito.testnet'
  }
]
```
